.github/workflows/build_and_deploy.yml

```
@@ -214,6 +220,54 @@ jobs:
           files: |-
             ${{ steps.build-voicevox-core-python-api.outputs.whl }}
           target_commitish: ${{ github.sha }}
+  build_xcframework:
+    needs: build_and_deploy
+    runs-on: macos-12
+    steps:
+      - name: Set ASSET_NAME env var
+        shell: bash
+        run: echo "ASSET_NAME=voicevox_core-xcframework-${{ env.VERSION }}" >> "$GITHUB_ENV"
```

あっ　xcframeworkってiOS限定じゃないんですね！
だとするとここの名称は、他のに合わせて`ios-xcframework-cpu`とかが良いのかなとか思いました。
（macOSでも使えるかと勘違いされる方を防げるかなと）
```suggestion
        run: echo "ASSET_NAME=voicevox_core-ios-xcframework-cpu-${{ env.VERSION }}" >> "$GITHUB_ENV"
```

onnxruntime-builderの方でも同じような命名規則にしたほうがいいかもです。
気づかずに申し訳ない。。 https://github.com/VOICEVOX/onnxruntime-builder/pull/19

---

確かにそうですね。
onnxruntime-builderにもPRを出しておきます。
386edf8 で対応しました。

---

@HyodaKazuaki すみません、気づかずPR作ってしまいました！！ 🙇 
https://github.com/VOICEVOX/onnxruntime-builder/pull/20#issue-1718167343